packages/flutter/test/painting/matrix_utils_test.dart

```diff
@@ -121,4 +121,79 @@ void main() {
       forcedOffset,
     );
   });
+
+  test('transformRect with no perspective (w = 1)', () {
+    const Rect l10t20r30b40 = Rect.fromLTRB(10, 20, 30, 40);
```

Comment:
nit: these names are a little hard to read and we try to avoid abbreviations.  Maybe this one can just be `tenByTwenty`, and the other one `fourtyByFourty`?

Alternatively, you could name them based on the expected inputs and outputs, even as simple as `inputRect` and `outputRect`.

Reply:
I called it rectangle20x20 and inlined the other rect in the one place it was used (as an expected answer for one of the tests). Or are words preferred to numbers?

Reply:
Numbers are fine, we just try to avoid abbreviations.